src/services/EnvironmentVariables.ts

```diff
@@ -88,4 +90,8 @@ export class EnvironmentVariables {
 		return this.RETURN_JOURNEY_URL;
 	}
 
+	sessionEventsTable(): any {
```

Comment:
We are missing the case for POST_EVENT_SERVICE to check for missing environment variables?